presubmit.py

```diff
@@ -13,10 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Presubmit script for fuzzbench."""
+# pylint: disable=wrong-import-position
+import os
+
+os.environ['FORCE_LOCAL'] = '1'
+if 'FORCE_NOT_LOCAL' in os.environ:
```

Comment:
should we throw an error instead of silently deleting the env variable? If a user passed that argument, they expect it to take effect, right?

Reply:
Yeah I just deleted that part.